packages/wrangler/src/index.tsx

```diff
@@ -702,6 +703,10 @@ export async function main(argv: string[]): Promise<void> {
           type: "number",
           default: 8787,
         })
+        .option("devtools-port", {
+          describe: "Port for devtools to connect to, defaults to 9229",
+          type: "number",
```

Comment:
The reason I don't want to do this is because I want the behavior to be like
- User specifies a port, then we commit to using that port
- The user doesn't specify a port, we try 9229, and failing that just pick one at random

If we set a "default option" then we unfortunately have no way at runtime to figure out if that arg was passed in by the user or selected for us by default, which prevents us from achieving this behavior

Reply:
Noted and agreed!